h2o-core/src/main/resources/www/login.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
Btw: a front-end guys should verify this one.

Furthermore, we should have un-protected resources (css, images) which you can access without authentication.